.planning/handovers/HDR_COLOR_FIX_PLAN.md

```
@@ -0,0 +1,643 @@
+        g: mix(rgb1.g, rgb2.g, size1, size2),
+        r: mix(rgb1.r, rgb2.r, size1, size2),
+    };
+}
+```
+
+### 2.3 Configurable peakNits
+
+**Files**: `engine/src/Options/Interfaces/IOptions.ts`, `engine/src/Options/Classes/Options.ts`
+
+Add option:
+```ts
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)